### PR TITLE
recommend using rollup and changesets for npm modules

### PR DESCRIPTION
**Pull request by sndrs**
update packages recommendations to use rollup and changesets

### CODE REVIEW

**Review by joecowton1 — APPROVED** [review]
Looks good!

File: npm-packages.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ Do not include or depend on [polyfills](https://developer.mozilla.org/en-US/docs
 
 #### Compiling
 
-Use the TypeScript compiler (`tsc`) to generate the JavaScript and TypeScript [declaration files](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) that you publish.
+Use [Rollup](https://rollupjs.org/guide/en/) to bundle you code.
```
Comment (tjsilver) [inline npm-packages.md:22]: you -> your?

**Review by SiAdcock — COMMENTED** [review]
Love it ❤️ 

There's a couple of recommendations from the [security review of changesets](https://docs.google.com/document/d/19NBdyxKJGSuo8VCwdq7KYxRwm1FaJXoR0cqT0i5jhfU/edit#heading=h.royzz2hqw13g) that it might be appropriate to mention here:

- To minimise the risk of exposing private information the changesets-bot should not be installed on private repositories. 
- The changesets-bot application has permissions to merge PRs through the ‘pull request write permission”. To mitigate this all repositories where the changesets-bot is installed should have branch protection enabled with “Require approvals” from Guardian staff before merging. 

What do you think?

**Comment by sndrs** [comment]
> To minimise the risk of exposing private information the changesets-bot should not be installed on private repositories

a github admin would need to install it, so i think this is ok

the value of branch protection kind of depends on the repo in question i think?

**Pull request merged**